schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl

```diff
@@ -130,18 +168,42 @@ It is a representation of the zygosity
 
     }
 
-    /**
-    This is the classification of the variant according to standard practice guidelines (e.g. ACMG)
-    */
-    // TODO: refactor with ACMGClassification in ExitQuestionnaire
-    enum VariantClassification {pathogenic_variant, likely_pathogenic_variant, variant_of_unknown_clinical_significance, likely_benign_variant,  benign_variant, not_assessed}
+    record GenePanel{
```

Comment:
Would it be such that the record GenePanel can be union {null, record} but all items within a record must be present (therefore you always get a GenePanel name and version in each record in the payload)

Reply:
Changed